package.json

```diff
@@ -30,7 +30,7 @@
     "seamless-immutable": "6.x"
   },
   "peerDependencies": {
-    "redux": "3.x"
+    "redux": "4.0.0-beta.2"
```

Comment:
Can we just do `>= 3.x <= 4` here? Will that allow consuming clients to pull in `4.0.0-beta.2`? I would think so.